tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/hive/HiveTableManager.java

```diff
@@ -83,9 +86,10 @@ public HiveTableManager(
 
     public HiveTableManager(
             QueryExecutor queryExecutor,
-            HdfsDataSourceWriter hdfsDataSourceWriter,
+            FileSystemDataSourceWriter fsDataSourceWriter,
             TableNameGenerator tableNameGenerator,
             HiveThriftClient hiveThriftClient,
+            String testDataBaseFSPrefix,
```

Comment:
should not this belong to `FileSystem` related things? IMO `HiveTableManager` should not be aware of this.